pytest_fixtures/api_fixtures.py

```diff
@@ -59,6 +63,14 @@ def module_manifest_org():
     return org
 
 
+@pytest.fixture(scope='module')
+def module_gt_manifest(module_org):
+    """Loads GT manifest in module org"""
+    manifest = manifests.clone(org_environment_access=True, name='golden_ticket')
+    manifests.upload_manifest_locked(module_org.id, manifest, interface=manifests.INTERFACE_CLI)
+    return manifest
```

Comment:
I have a concern about reusing the same organization created at this module level. This could be alleviated with a new organization created in this fixture.

Also, would returning the manifest or the organization be more appropriate? I see one test uses the manifest object for rct. However, most other uses of this fixture also require pulling in module org. This makes me think that returning the organization should be the primary behavior.

Reply:
> I have a concern about reusing the same organization created at this module level. This could be alleviated with a new organization created in this fixture.
> 
Creating a new org does mean I cannot use module_lce, module_ak_with_cv, and module_promoted_cv  as they all use module_org

Reply:
Changes made